sdk/confidentialledger/azure-confidentialledger/MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include _meta.json
+include *.md
+include azure/__init__.py
+recursive-include tests *.py
+recursive-include samples *.py
```

Comment:
what is `_meta.json`?

Also I think you want to add 
`include azure/ai/textanalytics/py.typed` (make sure there's an empty py.typed file like [this](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/py.typed) so you're released package has mypy support

Reply:
Removed `_meta.json`, and added `py.typed`.